### ISSUE TITLE
fix: no channels exist on source chain: %!w(<nil>)

### ISSUE THREAD

**Issue by pgoos** [dymensionxyz/e2e-tests#49]
Sometimes e2e tests are failing on
```
channel, err := ibc.GetTransferChannel(ctx, r, eRep, rollapp1.Config().ChainID, dymension.Config().ChainID)
require.NoError(t, err)
```
with error `no channels exist on source chain: %!w(<nil>)`
Example workflow run https://github.com/dymensionxyz/e2e-tests/actions/runs/8325457478/job/22779139434?pr=22
This error can randomly occur.

**Comment by pgoos** [comment]
Any progress?

**Comment by pgoos** [comment]
Related issue
```
Error:      	"[{STATE_OPEN ORDER_UNORDERED {transfer channel-0} [connection-0] ics20-1 transfer channel-0}]" should have 2 item(s), but has 1
```
would be great to fix both as part of this ticket

**Comment by trinitys7** [comment]
close as resolved in #64
